fabric/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,28 @@
+{
+  "schemaVersion": 1,
+  "id": "lamp",
+  "version": "${version}",
+
+  "name": "Lamp",
+  "description": "A modern annotations-driven commands framework for Java and Kotlin",
+  "authors": [
+    "Revxrsal",
+    "Camper_Samu"
+  ],
+  "contact": {
+    "homepage": "https://github.com/GrimAnticheat/Grim",
+    "sources": "https://github.com/GrimAnticheat/Grim/issues"
```

Comment:
Must be changed

Reply:
I forgot to change this while testing

Reply:
Addressed in [172c6d1](https://github.com/Revxrsal/Lamp/pull/118/commits/172c6d114fe320a27021c0114024f8a60b92c19c).